Component/inline code (#2642)
twilio-labs/paste: packages/paste-website/src/pages/components/inline-code/index.mdx

          name
          description
          status
          version
        }
      }
    }
    mdx(frontmatter: {slug: {eq: "/components/inline-code/"}}) {
      fileAbsolutePath
      frontmatter {
        slug
        title
      }
      headings {
        depth
        value
      }
    }
    allAirtable(filter: {data: {Feature: {eq: "inline-code"}}}) {
      edges {
        node {
          data {
            Documentation
            Figma
            Design_committee_review
            Engineer_committee_review
            Code
            status
          }
        }
      }
    }
  }
`;

<NormalizedComponentHeader
  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
  githubUrl="https://github.com/twilio-labs/paste/blob/main/packages/paste-core/components/inline-code/"
  storybookUrl="/?path=/story/components-inline-code--variants"
  description={props.data.mdx.frontmatter.description}
  data={props.data}
/>

---

<contentwrapper>

<PageAside data={props.data.mdx} />

<content>
  <LivePreview scope={{InlineCode}} language="jsx">
    {`<InlineCode>
    console.log(mySustainabilityGoals)
</InlineCode>`}
  </LivePreview>

## Guidelines

### About Inline Code

> The Inline Code component emphasizes short, inline pieces of computer code such as variable names, commands and function calls.

Reviewer: ```suggestion
The Inline Code component emphasizes short, inline pieces of computer code such as variable names, commands, and function calls.
```

> The Inline Code component emphasizes short, inline pieces of computer code such as variable names, commands and function calls.
> It is also used to display important inline code snippets, such as SIDs, IDs and API keys

Reviewer: ```suggestion
The Inline Code component emphasizes short, inline pieces of computer code such as variable names, commands, and function calls.
It's also used to display important inline code snippets, such as SIDs, IDs, and API keys
```
